docs/rest-api/client.md

```diff
@@ -324,11 +336,16 @@ You can disable/enable a client using the API. The following parameters listed b
         <div id="tab-1" class="tab__content">
 <pre>
 ```
-  curl -X PUT -H "App-Id: {app-id}" -H "App-Key: {app-key}" -H "Auth-Token: {auth-token}"
-  -H "Content-Type: application/json"
-  -H "Cache-Control: no-cache"
-  -d '{ "id": 123456, "disable": true}'
-"https://{host}.bookingbug.com/api/v1/admin/{company_id}/client"
+  curl -X PUT \
+   https://{host}.bookingbug.com/api/v1/admin/{company_id}/client \
+  -H 'App-Id: {app-id}' \
+  -H 'App-Key: {app-key}' \ 
+  -H 'Auth-Token: {auth-token}' \
+  -H 'Content-Type: application/json' \
+  -H 'Cache-Control: no-cache' \
+  -d '{
+   "id": 123456, "disable": true
```

Comment:
Strange the we need to put the id into the payload and not in the URL as with many other business entities.
PUT /admin/{company_id}/client/{Client_id} ?